gcp_variant_transforms/libs/header_merger.py

```diff
@@ -12,23 +12,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""beam combiner function for merging VCF file headers."""
+"""class for merging 2 VcfHeader objects"""
```

Comment:
nit: s/class/Class. Also add `.` at the end of the sentence.

Reply:
Done.